<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F.Y.D. - Phantom texture rendering application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StartApp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Program</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>Application status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause_QB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>241</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause and get QB info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowImage">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Show webcam image</string>
    </property>
   </widget>
   <widget class="QLabel" name="IP_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F.Y.D. connection mode:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="SavePhantomButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Phantom Log</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>IMAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveAccelerationsButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Accelerations Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SeeOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>See Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="increase_kvib">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>324</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrease_kvib">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>324</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vibration_gain_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>334</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vibration Gain:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vibration_gain_value_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="XY_SF">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="PWM_method_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vibration PWM method:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PWM_method_combo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DFT321 (DFT 3-to-1)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SoC (Sum of Components)</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="InfoTextEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string>Application Information</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="XY_SF_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>XY Scale Factor</string>
    </property>
   </widget>
   <widget class="QLabel" name="XY_SF_value_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 mm FYD -&gt; x mm robot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Vibration_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Vibration enabled</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="Connection_mode_combo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ethernet network (on 192.168.0.100)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FYD Touch Wireless network (on 10.42.0.1)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Help">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>520</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QSlider" name="motor_ref0">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="singleStep">
     <number>500</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="motor_ref1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="singleStep">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="motor0_ref_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor 1 Reference:</string>
    </property>
   </widget>
   <widget class="QLabel" name="motor1_ref_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor 2 Reference:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Motor_ref_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Change motor reference position</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="motor0_ref_value_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="motor1_ref_value_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrease_kp0">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="increase_kp0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrease_kp2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="increase_kp2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="kp0_value_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="kp2_value_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vibration_gain_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal movement Gain:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vibration_gain_label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical movement Gain:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
